includes/class-sensei-course-structure.php

```diff
@@ -854,4 +862,39 @@ public function get_first_incomplete_lesson_id() {
 
 		return false;
 	}
+
+	/**
+	 * Get module for teacher by slug.
+	 *
+	 * @since x.x.x
+	 *
+	 * @param string $slug             The probable slug.
+	 * @param array  $additional_slugs Additional slugs to search with.
+	 *
+	 * @return WP_Term|null The found term or null.
+	 */
+	private function get_module_for_teacher_by_slug( string $slug, array $additional_slugs = [] ) {
```

Comment:
Is the `$additional_slugs` actually the newly generated slug by the term title? If this is the case, what do you think about passing the `$title` instead of `$additional_slugs` and renaming the method to `get_module_for_teacher_by_slug_or_title( $slug, $title )`. After that, you could generate the title slug ( `$this->get_module_slug( $title )` ) inside the method which will make the logic a bit more clear.